prisma/schema.prisma

```diff
@@ -245,6 +245,7 @@ model WalletLog {
   wallet    WalletType
   level     LogLevel
   message   String
+  context   Json?      @db.JsonB
```

Comment:
I'm curious why you used JSON here? Does context vary a lot or do you anticipate that it will?

Reply:
I anticipated that we might want to add more to it and I didn't think that relationships would be important for logs. If a log has a wrong relationship because the database didn't enforce it, it will lose relevance over time because logs become less relevant over time.

For example, we could add `invoiceId` and don't have to care about migrating previous logs in the database with JSON. The client will have to care about old logs for a while until we delete them. We don't yet but this is what I anticipated we will do at some point.